model/host/host.go

```diff
@@ -325,7 +325,7 @@ const (
 	// Max number of spawn hosts with no expiration for user
 	DefaultUnexpirableHostsPerUser = 1
 	// Max total EBS volume size for user
-	DefaultMaxVolumeSizePerUser = 200
+	DefaultMaxVolumeSizePerUser = 500
```

Comment:
No--the amount that a user can make over all their volumes is defined in the admin settings (I changed this to be 700 in prod). So assuming that they have enough space left, their new volume will default to showing 500 GiB as the volume size, otherwise however much space they have left, so the 500 is just a suggested default and imposes no restrictions on a user.

Reply:
It looks like this value is only used [here](https://github.com/evergreen-ci/evergreen/blob/e77c290a01bbd0966731e1540ef1415ac0d9fc98/rest/model/admin.go#L1455)

Reply:
Oh sorry haha yeah I actually didn't mean to change this, although I'm going to keep it:  this value is still just the default max volume size per user, and it's better than 200 given what you said earlier. But we update it from the config later in the file. 
https://github.com/evergreen-ci/evergreen/blame/e77c290a01bbd0966731e1540ef1415ac0d9fc98/rest/model/admin.go#L1502